### PR TITLE
feat: JSR support

### PR DESCRIPTION
**Pull request by iuioiua**
I'm unsure why Deploy fails to detect the `deno.json` file. Once this PR merges, the publish workflow should publish the module to JSR.

### CODE REVIEW

File: .github/_typos.toml
Comment (iuioiua) [inline .github/_typos.toml:1]: Because `vendor/` has typos.

File: README.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ This is required for OAuth solutions that span more than one sub-domain.
 
 1. Create your OAuth configuration and Fresh plugin.
 
-   ```ts
+   ```ts, ignore
```
Comment (iuioiua) [inline README.md:228]: Because the snippet uses the `$fresh/serve.ts` import specifier is being used. Note: Fresh isn't available on JSR yet.

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by kt3k** [comment]
I'm not completely sure this type of vendoring works with `deno publish` out of the box though

**Comment by iuioiua** [comment]
> I'm not completely sure this type of vendoring works with `deno publish` out of the box though

How so? I used `deno vendor` and tweaked things until `deno task ok` and `deno publish --dry-run` both passed.

**Comment by kt3k** [comment]
I just didn't know the vendoring example in jsr. It might be just fine. Let's try.

I guess the issue of Deno Deploy might be resolved by adding `"@std/http/": "jsr:@std/http@^0.220.0/",` to the import map?

**Comment by iuioiua** [comment]
For reference, the official JSR documentation suggests this as a means of migrating from /x (see [here](https://jsr.io/docs/migrate-x-to-jsr#1-refactor-away-from-https-imports)). The issue is that the `deno.json` file isn't being automatically detected. So it's not even getting to the point of re-mapping import specifiers. Yusuke said this is because Deploy is yet to be updated, but he will let me know once it is done.

**Pull request merged**